src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -129,7 +129,7 @@
   float: right;
   height: $height;
   padding: 0 $indent;
-  margin-left: -100%;
+  margin-left: $indent;
```

Comment:
@jan-capiak 
Thanks for your PR.

May I ask your to provide positive test for firefox, and mobile on iOS and Android.
The point is that some firefox for example does not support `text-indent` at styled window-objects.

Reply:
Furthermore the icon is not meant to consume space, now it takes up the size of `$indent`

Reply:
And it shall overlay any browser-intrinsic dropdown icon, which can't be always removed by CSS.

Reply:
The overflow of `<select/>` is made intentionally for cross browser support.

<img width="345" alt="screen shot 2018-06-18 at 15 21 02" src="https://user-images.githubusercontent.com/914443/41538252-272ef772-730b-11e8-9e32-2cb10ac1921b.png">